### ISSUE TITLE
return nyquist velocity and maximum unambiguous range from radar data

### ISSUE THREAD

**Issue by swnesbitt** [Unidata/thredds#414]
Hello,

I am trying to access NEXRAD level 2 data through the S3 archive hosted by Unidata, and noted that the Nyquist velocity and maximum unambiguous range are stored in the raw files (dimensions of time), but not in the OPENDAP dataset stream.

Is there a tool to get at this information from the OPENDAP stream without downloading the whole file?

Thanks,
-Steve

-- 
Steve Nesbitt, Associate Professor
Department of Atmospheric Sciences, University of Illinois at Urbana-Champaign
105 S. Gregory St., MC223, Urbana, IL 61801-3070  USA
voice : +1.217.244.3740  fax : +1.217.244.1752
internet : snesbitt@illinois.edu ; https://www.atmos.illinois.edu/people/snesbitt ; http://publish.illinois.edu/snesbitt

**Comment by dopplershift** [comment]
No, there is no way to do it without downloading the file. I agree (as a radar guy) that this is a pretty big oversight.

**Comment by swnesbitt** [comment]
Not familiar with the back end of the thredds server, but could this be
included in future versions?

On Monday, February 8, 2016, Ryan May <notifications@github.com> wrote:

> No, there is no way to do it without downloading the file. I agree (as a
> radar guy) that this is a pretty big oversight.
>
> —
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Unidata_thredds_issues_414-23issuecomment-2D181630206&d=BQMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=oh_IyBh3p6AhkrNBZyzmmDfeXg2WKrzfthPLOoHreWM&m=aez8OTlAN77YSxmH6_yH6z5WcW-zjODdw4d71Gk0I4s&s=5i5nPlb_xAgz1BfH4dhH_-shn11sWACskpHZjexPLwA&e=>
> .
>


-- 
Steve Nesbitt, Associate Professor
Department of Atmospheric Sciences, University of Illinois at
Urbana-Champaign
105 S. Gregory St., MC223, Urbana, IL 61801-3070  USA
voice : +1.217.244.3740  fax : +1.217.244.1752
internet : snesbitt@illinois.edu ;
https://www.atmos.illinois.edu/people/snesbitt ;
http://publish.illinois.edu/snesbitt

**Comment by dopplershift** [comment]
Absolutely. It'd go into our 5.x branch which is a major new version which should see its first alpha/beta release soon. Thus I hesitate to give an ETA on when it might be done.
